wiki/installing-and-updating-drivers.md

```diff
@@ -0,0 +1,164 @@
+# Installing and Updating Drivers 
+In this guide, you will learn how to download or update drivers on your computer.
+
+::: warning Warning
+It is highly recommended to avoid using third-party software to update or install your drivers.
+:::
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly identify the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager. You can access the device manager by right-clicking the Windows start button or by pressing <kbd>Win</kbd> + <kbd>X</kbd> and selecting Device Manager. In the Device Manager, look for the device you require drivers for, and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you require drivers for.
+
+## Method 1: Using Windows Update
+You can usually find drivers for your devices in Windows Update, instead of having to manually download the file.
+
+1. Open Settings through the start menu or by pressing <kbd>Win</kbd> + <kbd>I</kbd>.
+ 
+2. Go to Update & security (on Windows 10) or Windows Update (on Windows 11).
+ 
+3. Click on “Check for Updates” or "Download & Install Updates".
+ 
+
+Windows Update might say that you are up-to-date, even if optional updates are available. You can check for additional optional updates in the 'Optional updates' section of Windows Update settings.
+
+On Windows 11:
+
+1. Click "Advanced options".
+
+<img src="./img/installing-and-updating-drivers/optional-updates/windows-update-advanced-options.png" alt="Windows 11's Windows Update settings, with the Advanced options button highlighted." width=500px>
+
+2. Click "Optional updates".
+
+<img src="./img/installing-and-updating-drivers/optional-updates/windows-update-advanced-options-optional-updates.png" alt="The Advanced options menu, with the Optional updates button highlighted." width=500px>
+
+3. Click "Driver updates" to show the list of available driver updates, if there are any.
+
+<img src="./img/installing-and-updating-drivers/optional-updates/windows-update-optional-updates-driver-updates.png" alt="The Optional updates menu, with the Driver updates menu highlighted." width=500px>
+
+4. Select any optional updates you wish to install, then click "Download & install".
+
+<img src="./img/installing-and-updating-drivers/optional-updates/windows-update-driver-update-example.png" alt="An available optional driver update, with the Download & install button selected." width=500px>
+
+On Windows 10, a "View optional updates" link will appear under "Check for updates" if optional updates are available. Click on it to see the list of available driver updates.
+
+## Method 2: Download using the manufacturer's website.
```

Comment:
```suggestion
## Method 2: Download using the manufacturer's website.
::: tip Note
Using a laptop, or a pre-built desktop? You should probably download drivers from your OEM's website instead. See [Method 3](#method-3-download-using-the-oem-s-website).
:::
```

Reply:
I don't see any difference

Reply:
I added a note under the Method 2 header.

Reply:
Ok...